frontend/src/components/place/PlacePreview.vue

```diff
@@ -157,6 +156,11 @@ export default {
             type: Number,
         }
     },
+    computed: {
+        photo: function () {
+            return this.place.photos[0].img_url ? this.place.photos[0].img_url : 'http://via.placeholder.com/128x128';
```

Comment:
Download this placeholder file http://via.placeholder.com/128x128 and move into assets dir